docs/Table of Contents.md

```diff
@@ -0,0 +1,62 @@
+# Terraform Provider Documentation
+
+This table of contents shows the resources for files in the /datasources path and the /resources path.
```

Comment:
How about "This table lists all of the available OCI resources and data sources."